main/mySpace/access_details_session.php

```diff
@@ -485,12 +486,14 @@ function getReport($userId, $from, $to, $addTime = false)
         $courseSessionTable = $result['second'];
         $totalCourseSessionTable = $result['third'];
         $total = $result['total'];
-        $tableList .= '<div style="text-align:center">'.Display::page_subheader2($dateToCheck).'</div>'.$table->toHtml();
+        $iconCalendar = Display::return_icon('calendar.png',null,[],ICON_SIZE_SMALL);
+        $tableList .= '<div class="date-calendar">'.Display::page_subheader2($iconCalendar.get_lang('Date').': '.$dateToCheck).'</div>';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 136 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)